snuba/datasets/storages/factory.py

```diff
@@ -23,7 +23,6 @@ class _StorageFactory(ConfigComponentFactory[Storage, StorageKey]):
     def __init__(self) -> None:
         with sentry_sdk.start_span(op="initialize", description="Storage Factory"):
             self._config_built_storages: dict[StorageKey, Storage] = {}
-            self._writable_storages: dict[StorageKey, Storage] = {}
```

Comment:
do you still need these?

```
 self._config_built_storages: dict[StorageKey, Storage] = {}
            self._dev_writable_storages: dict[StorageKey, Storage] = {}
            self._cdc_storages: dict[StorageKey, Storage] = {}
            self._dev_cdc_storages: dict[StorageKey, Storage] = {}
            self._non_writable_storages: dict[StorageKey, Storage] = {}
            self._dev_non_writable_storages: dict[StorageKey, Storage] = {}
```

Reply:
Yes, to `_config_built_storages` because errors storage has not been move to config yet. I don't know have context behind what all the other ones do but they are empty so I think they can be removed? What do you think?

Reply:
Yeah I think they can go, they're not used anywhere

Reply:
`self._cdc_storages` doesn't need to exist outside of the `__initialize` function scope.